lib/delayed/command.rb

```diff
@@ -120,7 +124,12 @@ def setup_pools
 
     def run_process(process_name, options = {})
       Delayed::Worker.before_fork
-      Daemons.run_proc(process_name, :dir => options[:pid_dir], :dir_mode => :normal, :monitor => @monitor, :ARGV => @args) do |*_args|
+      Daemons.run_proc(process_name,
+                       :dir => options[:pid_dir],
+                       :dir_mode => :normal,
+                       :monitor => @monitor,
+                       :ARGV => @args,
+                       :pid_delimiter => options[:pid_delimiter]) do |*_args|
```

Comment:
I think this isn't backwards compatible with older daemons versions, as the options parsing will raise an error when it encounters this unknown option. Ideally there would be a PR that doesn't pass this option to daemons versions that don't support it.